pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -113,13 +104,14 @@ parameter_types! {
     // Settlement:
     pub const MaxLegsInInstruction: u32 = 10;
 
-    // Offences:
-    pub OffencesWeightSoftLimit: Weight = Perbill::from_percent(60) * MaximumBlockWeight::get();
-
     // I'm online:
-    pub const SessionDuration: BlockNumber = EPOCH_DURATION_IN_SLOTS as _;
     pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
 
+    pub const MaxAuthorities: u32 = 100_000;
```

Comment:
Do you know what these constants relate to - did we get the default values from upstream?

Reply:
So these are documented here:
- https://github.com/paritytech/substrate/blob/f318350b6410ef47e438aae24c0b4779373d0a7b/frame/authority-discovery/src/lib.rs#L48
- https://github.com/paritytech/substrate/blob/f318350b6410ef47e438aae24c0b4779373d0a7b/frame/babe/src/lib.rs#L177
- https://github.com/paritytech/substrate/blob/f318350b6410ef47e438aae24c0b4779373d0a7b/frame/grandpa/src/lib.rs#L127

And their values come from polkadot / kusama / ...:
- https://github.com/paritytech/polkadot/blob/594e3b95516693f807f512e0deb26bad2d4f988d/runtime/polkadot/src/lib.rs#L788-L791